### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by deepu105**
Add more logos to stack on home screen
Add list of companies and projects contributing to jhipster
Optimize image size using `gulp images`
polish

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -340,14 +346,34 @@ <h2 class="heading">Server side options</h2>
...
+                <li class="clip img-circle new">
+                    <img src="img/icons/swagger.png"/>
+                    <span class="title_sub_tech">Swagger</span>
+                </li>
+                <li class="clip img-circle new">
+                    <img src="img/svg/docker.svg"/>
+                    <span class="title_sub_tech">Docker</span>
+                </li>
+                <li class="clip img-circle new">
+                    <img src="img/svg/kubernets.svg"/>
```
Comment (jdubois) [inline index.html]: why not "kubernetes"? The name of file is wrong?

File: pages/thanks.md [hunk 2]
```diff
@@ -22,3 +21,17 @@ Those companies help us to develop JHipster
...
+
+# <i class="fa fa-gift"></i> Companies and projects contributing to JHipster
+
+JHipster has got a lot of [contributors](https://github.com/jhipster/generator-jhipster/graphs/contributors), and there are official contributions from popular projects and companies as well. Below is the list of companies and projects contributing to JHipster and we are very thankful for their contributions.
+
+*   [Ippon Technologies](http://www.ippon.fr/), which provides time and hardware to [@juliendubois](https://twitter.com/juliendubois) so he can improve the project
+*   [XebiaLabs](https://xebialabs.com/), which provides time and hardware to [@deepu105](https://twitter.com/deepu105) so he can improve the project
+*   [Google](https://google.com) contributed the Kubernetes(http://kubernetes.io/) sub generator. It is maintained by [Ray Tsang](https://twitter.com/saturnism)
+*   [Heroku](https://www.heroku.com/) contributed the Heroku sub generator. It is maintained by [Joe Kutner](https://twitter.com/codefinger)
```
Comment (mraible) [inline pages/thanks.md]: Any chance of getting some Stormpath love here? ;)

* [Stormpath](https://stormpath.com/) contributed the Stormpath sub generator. It is maintained by [Matt Raible](https://twitter.com/mraible)

Comment (jdubois) [inline pages/thanks.md]: Yes of course! The list should be improved (I need to ask people from ehcache, etc.), and I would even put it in a more visible place.

File: index.html (hunk 1 above)
Comment (deepu105) [inline index.html]: the file name is wrong

File: pages/thanks.md (hunk 2 above)
Comment (deepu105) [inline pages/thanks.md]: I thought of adding a companies-contributing page and then referencing it right after the companies using section in home screen then I thought you might not like it 😸

Comment (deepu105) [inline pages/thanks.md]: I think we could mention more like Boxfuse for official support for JHipster etc

Comment (jdubois) [inline pages/thanks.md]: Yes I'm adding Boxfuse, I'm sure @axelfontaine will be happy about it

Comment (axelfontaine) [inline pages/thanks.md]: @deepu105 @jdubois Thank you very much guys! Let me know if I can help.

**Comment by deepu105** [comment]
sure go ahead


Thanks & Regards,
Deepu

**Pull request merged**